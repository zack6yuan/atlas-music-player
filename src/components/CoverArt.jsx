import placeholder from "../assets/placeholder.svg"

export default function CoverArt() {
    return (
        <div className="cover-art 2xl:scale-33 2xl:-mt-95 2xl:-ml-66 xl:scale-33 xl:-mt-95 xl:-ml-85 lg:scale-33 lg:-mt-94 lg:-ml-100 md:scale-29 md:-mt-88 md:-ml-88">
            <img src={ placeholder } alt="" className="rounded-3xl"/>
        </div>
    )
}