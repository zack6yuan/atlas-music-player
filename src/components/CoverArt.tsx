import placeholder from "../assets/placeholder.svg"

export default function CoverArt() {
    return (
        <div className="cover-art 2xl:scale-60 2xl:-mt-30 2xl:-mr-30 xl:scale-90 xl:mt-1 xl:ml-33 lg:scale-80 lg:-mt-6 lg:ml-3 md:scale-85 md:-mt-2 md:-ml-1 sm:scale-92">
            <img src={ placeholder } alt="" className="rounded-xl"/>
        </div>
    )
}