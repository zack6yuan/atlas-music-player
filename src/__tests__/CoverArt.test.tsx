import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import CoverArt from "../components/CoverArt"

test("Cover Art renders correctly", () => {
    const { container } = render(
        <CoverArt />
    );
    expect(container).toMatchSnapshot();
});