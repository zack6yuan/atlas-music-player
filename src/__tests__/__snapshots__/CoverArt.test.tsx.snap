// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Cover Art renders correctly 1`] = `
<div>
  <div
    class="cover-art 2xl:scale-39 xl:scale-120 xl:mt-13 xl:ml-14 xl:mr-13 lg:scale-90 lg:-mt-0 lg:ml-0 md:scale-85 md:-mt-2 md:-ml-1 sm:scale-92"
  >
    <img
      alt=""
      class="rounded-xl"
      src="https://utfs.io/f/E9fJnaKtTy1bOclGsk1hB7xMLwUVFDiXypZukQrcnYbgdEv6"
    />
  </div>
</div>
`;
